src/data/multiset/basic.lean

```diff
@@ -1746,6 +1746,21 @@ by { simp_rw [←filter_eq', eq_comm], congr }
 
 end
 
+lemma card_filter_eq_count (s : multiset α) (p : α → Prop) [decidable_pred p] :
+  (s.filter p).card = (s.map p).count true :=
```

Comment:
Sounds like this lemma should be a simp lemma the other way around, much like `list.countp_true`.